vignettes/articles/05-read-the-outputs.Rmd

```diff
@@ -17,58 +17,106 @@ proceeds to explain in depth how to read the data in each of the files.
 
 ## Overview
 
-There are eight output files that the user can expect when running the stress
-test for a given set of parameters.
+The source code generates nine result (output) files after completing the 
+stress test for a given set of parameters. They principally contain estimates of the
+market risk of companies in the portfolio and of the credit risk of said companies
+in a corporate loan book. The source code additionally outputs a file with 
+company-level scenario production trajectories and log file which holds 
+information on the settings of current the stress test run.
 
-They can loosely be grouped in files that contain estimates of the market risk
-of companies in the portfolio and of the credit risk of said companies in a
-corporate loan book. Depending on the use case, one or the other may contain
-more relevant information. In addition, a log file is written to the results
-directory. It holds an overview on holdings that could not be considered in the
-analysis.
+**Log file**
 
-### Market Risk
+* log_file_\<suffix\>.txt
+
+**Production trajectory data**
```

Comment:
Maybe we can call it something like `future cash flow trajectory data`?
In the data base version, we will not be able to share the production data anymore, just the financial results. Hence this could already hint to what we are trying to focus on here. This would also highlight that FIs can use this file to plug the discounted cash flows into their own models. That would seem less straight forward with production trajectories. What do you think?